<throwable>
  <class>org.frankframework.core.SenderException</class>
  <message>exception processing message using request processor [ibis4test-ForEachChildElementPipe]: (RequestProcessorException) RequestProcessor [ibis4test-ForEachChildElementPipe] caught exception: Pipe [ForEachChildElementPipe_Record] caught exception: (SaxException) Could not process list of elements using xpath [requests/request]: (TransformerException): (SaxException): exitState=ERROR</message>
  <cause>
    <throwable>
      <class>nl.nn.adapterframework.dispatcher.RequestProcessorException</class>
      <message>RequestProcessor [ibis4test-ForEachChildElementPipe] caught exception</message>
      <cause>
        <throwable>
          <class>org.frankframework.core.ListenerException</class>
          <message>Pipe [ForEachChildElementPipe_Record] caught exception: (SaxException) Could not process list of elements using xpath [requests/request]: (TransformerException): (SaxException): exitState=ERROR</message>
          <cause>
            <throwable>
              <class>org.frankframework.core.PipeRunException</class>
              <message>Pipe [ForEachChildElementPipe_Record] caught exception: (SaxException) Could not process list of elements using xpath [requests/request]: (TransformerException): (SaxException): exitState=ERROR</message>
              <cause>
                <throwable>
                  <class>org.frankframework.core.SenderException</class>
                  <message>(SaxException) Could not process list of elements using xpath [requests/request]: (TransformerException): (SaxException): exitState=ERROR</message>
                  <cause>
                    <throwable>
                      <class>org.frankframework.xml.SaxException</class>
                      <message>Could not process list of elements using xpath [requests/request]</message>
                      <cause>
                        <throwable>
                          <class>javax.xml.transform.TransformerException</class>
                          <message>org.frankframework.xml.SaxException: exitState=ERROR</message>
                          <cause>
                            <throwable>
                              <class>org.frankframework.xml.SaxException</class>
                              <message>exitState=ERROR</message>
                              <cause>
                                <throwable>
                                  <class>org.frankframework.core.SenderException</class>
                                  <message>exitState=ERROR</message>
                                </throwable>
                              </cause>
                            </throwable>
                          </cause>
                        </throwable>
                      </cause>
                    </throwable>
                  </cause>
                </throwable>
              </cause>
            </throwable>
          </cause>
        </throwable>
      </cause>
    </throwable>
  </cause>
</throwable>
